README.md

```diff
@@ -171,6 +172,30 @@ Under `scenario_params_list`, enter a list of sets of scenario(s). Each set star
 
 ![Example graphs](./img/results_graphs_example.png)
 
+### Reference scenarios
+
+#### Description of the reference scenarios
+
+We defined 5 reference scenarios on which we propose to test and benchmark the different multi-partner learning approaches and contributivity measurement methods.
+
+The 5 reference scenarios are described on the following schema ([link](https://docs.google.com/drawings/d/1_ovwpBTx_Rno5FO1AcODM7cJ6a_EPaBOh9OYl952J-4/edit?usp=sharing) to editable version):
+
+![Reference scenarios](./img/reference_scenarios.png)
+
+In brief:
+
+- Scenarios 1 and 2 with 2 partners only enable simple baselines, with different data splits (each partner having samples of different classes in the first scenario, of all classes both in the second one), and introducing corrupted data;
+
+- Scenario 3 proposes a more realistic configuration, with partners having both samples of common classes and samples of different classes each;
+
+- Scenario 4 offers a case with 5 partners and an identical distribution of data samples of all classes, but with 1 partner having its data entirely corrupted;
+
+- Scenario 5 is more complex, with 11 partners, and a mix of identical distribution for some partners, different one for others, and corrupted data also.
```

Comment:
I think this description is unclear. 
Maybe : the samples of each class are distributed among various subset of partners, of which some overlap. 
Not sure that easier, but I think something is missing in the reference to identical distribution/different one

Reply:
OK, clarified now